examples/advanced.ts

```diff
@@ -47,67 +48,75 @@ async function createCacheExample() {
   }
 }
 
 async function setGetDeleteExample() {
   // ttl is an optional field on most write operations, but you can provide it if you
   // want to override the default ttl that you specified when constructing your client.
   const exampleTtlSeconds = 10;
-  console.log(
+  logger.info(
     `Storing key=${cacheKey}, value=${cacheValue}, ttl=${exampleTtlSeconds}`
   );
   const setResponse = await momento.set(cacheName, cacheKey, cacheValue, {
     ttl: exampleTtlSeconds,
   });
   if (setResponse instanceof CacheSet.Success) {
-    console.log('Key stored successfully!');
+    logger.info('Key stored successfully!');
   } else if (setResponse instanceof CacheSet.Error) {
-    console.log(`Error setting key: ${setResponse.message()}`);
+    logger.info(`Error setting key: ${setResponse.message()}`);
+  } else {
+    throw new Error(`Unrecognized response: ${setResponse.toString()}`);
   }
 
   const getResponse = await momento.get(cacheName, cacheKey);
   if (getResponse instanceof CacheGet.Hit) {
-    console.log(`cache hit: ${String(getResponse.valueString())}`);
+    logger.info(`cache hit: ${String(getResponse.valueString())}`);
   } else if (getResponse instanceof CacheGet.Miss) {
-    console.log('cache miss');
+    logger.info('cache miss');
   } else if (getResponse instanceof CacheGet.Error) {
-    console.log(`Error: ${getResponse.message()}`);
+    logger.info(`Error: ${getResponse.message()}`);
+  } else {
+    throw new Error(`Unrecognized response: ${getResponse.toString()}`);
   }
 
   const deleteResponse = await momento.delete(cacheName, cacheKey);
   if (deleteResponse instanceof CacheDelete.Error) {
-    console.log(`Error deleting cache key: ${deleteResponse.message()}`);
+    logger.info(`Error deleting cache key: ${deleteResponse.message()}`);
   } else if (deleteResponse instanceof CacheDelete.Success) {
-    console.log('Deleted key from cache');
+    logger.info('Deleted key from cache');
+  } else {
+    throw new Error(`Unrecognized response: ${deleteResponse.toString()}`);
   }
 }
 
 async function concurrentGetsExample() {
-  console.log('Saving 10 values to cache');
+  logger.info('Saving 10 values to cache');
   for (let i = 1; i <= 10; i++) {
     await momento.set(cacheName, `key${i}`, `value${i}`);
   }
-  console.log('Initiating 10 concurrent gets');
+  logger.info('Initiating 10 concurrent gets');
   const getPromises = range(10).map(i => momento.get(cacheName, `key${i + 1}`));
   const getResponses = await Promise.all(getPromises);
   getResponses.forEach((response, index) => {
     const key = `key${index + 1}`;
     if (response instanceof CacheGet.Hit) {
-      console.log(
+      logger.info(
         `Concurrent get for ${key} returned ${response.valueString()}`
       );
     } else {
-      console.log(
+      logger.info(
         `Something went wrong with concurrent get for key ${key}: ${response.toString()}`
       );
     }
```

Comment:
Not a blocker but do we want to handle the error case separately from an unknown response?

Reply:
I kind of wanted to illustrate that the user can choose which level they want to handle it at by including some examples of both.  wdyt?

Reply:
Sounds good. Just wanted to make sure it wasn't an oversight